mender-convert

```diff
@@ -723,27 +723,41 @@ sudo true
 case "$1" in
   raw-disk-image-shrink-rootfs)
     do_raw_disk_image_shrink_rootfs || rc=$?
-  [[ $rc -ne 0 ]] && { return 1; }
+    [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
```

Comment:
The rootfs partition in the raw disk image has been shrinked successfully!
You can now convert the output raw disk image ($raw_disk_image) to a Mender disk image.